library/src/main/java/com/anthonycr/grant/PermissionsManager.java

```diff
@@ -176,6 +176,18 @@ public static boolean hasAllPermissions(@Nullable Context context, @NonNull Stri
     }
 
     /**
+     * This static method can be used to check whether or not you should show request permission rational.
+     * If you pass in a null Activity object, it will return true
+     * @param activity the Activity necessary to check the permission
+     * @param permission the permission to check
+     * @return false if user selected never ask permission again, true otherwise
+     */
+    @SuppressWarnings("unused")
+    public static boolean shouldShowRequestPermissionRationale(@Nullable Activity activity, @NonNull String permission) {
+        return activity == null || ActivityCompat.shouldShowRequestPermissionRationale(activity, permission);
```

Comment:
I think this should be `activity != null && ActivityCompat...` because if the activity is null (let's assume we are in a Fragment detached from its parent activity), then actually requesting the permission will fail as it also requires an activity, which means we don't care about showing the rationale because we won't be able to show the permissions request dialog.